packages/terra-date-time-picker/tests/wdio/date-time-picker-spec.js

```diff
@@ -691,4 +691,34 @@ Terra.describeViewports('DateTimePicker', ['tiny', 'large'], () => {
 
     Terra.it.validatesElement();
   });
+
+  describe('With timezone', () => {
+    before(() => browser.url('/#/raw/tests/terra-date-time-picker/date-time-picker/date-time-picker-with-timezone'));
+
+    Terra.it.matchesScreenshot();
+  });
+
+  describe('With timezone missing hour', () => {
+    before(() => {
+      browser.url('/#/raw/tests/terra-date-time-picker/date-time-picker/date-time-picker-with-timezone-missing-hour');
+      browser.refresh();
+      Terra.hideInputCaret('input[name="terra-time-minute-input"]');
```

Comment:
This should happen automatically by terra-toolkit

Reply:
removed in this commit 2795049548a9f35213a3e290861bea40d3ff2ffe